modules/core/keeper/sdk_middleware.go

```diff
@@ -0,0 +1,129 @@
+package keeper
```

Comment:
I don't have a strong preference on 
`modules/core/sdk_middleware/sdk_middleware.go`
vs
`modules/core/keeper/sdk_middleware.go`

but my primary concern is I want to clearly differentiate the difference between IBC middleware and a transaction handler added into the SDK middleware stack

Reply:
Prefer this approach as well :)